cmd/clustersynchro-manager/app/options/options.go

```diff
@@ -90,6 +91,10 @@ func (o *Options) Flags() cliflag.NamedFlagSets {
 	genericfs.Float32Var(&o.ClientConnection.QPS, "kube-api-qps", o.ClientConnection.QPS, "QPS to use while talking with kubernetes apiserver.")
 	genericfs.Int32Var(&o.ClientConnection.Burst, "kube-api-burst", o.ClientConnection.Burst, "Burst to use while talking with kubernetes apiserver.")
 	genericfs.IntVar(&o.WorkerNumber, "worker-number", o.WorkerNumber, "The number of worker goroutines.")
+	genericfs.StringVar(&o.ShardingName, "sharding-name", o.ShardingName, "The name of the sharding that the manager is to synchro.")
+	if o.ShardingName != "" {
```

Comment:
o.shardingName must be "" when Flags() is called, so this logic won't be handled